content/report_RHF.tex

```diff
@@ -1,9 +1,244 @@
+ \label{subsec:intro}
+ Before we actually dive into the code, we will discuss the theory behind it.
+ The actual goal of this experiment is to calculate the energy of a molecule,
+ mainly to solve the Schrödinger equation for that molecule. This, unfortunately,
+ is impossible. So we have to find an alternative. This alternative would be
+Hartree-Fock theory. Bluntly put we can define another operator, the Fock operator,
+that consists of one-electron operators and two-electron operators. This is 
+displayed in Equation \ref{eq: fock matrix}. 
+
+\begin{equation}\label{eq: fock matrix}
+    \hat{f}(1) = \hat{h}(1) + 2\sum_i^{N/2}\hat{J}_i - \sum_i^{N/2}\hat{K}_i
+\end{equation}
+ J is the Coulomb operator, it accounts for the interelectronic repulsion. K is 
+ exchange operator, that accounts for the stabilizing interaction between 
+ electrons that have the same spin. 
+ \subsection{Deriving the Fock Operator}
+ \label{subsec:fock matrix}
+Let us step back a bit here to see where this operator comes from. To do that, we 
+need to begin with the true Hamiltonian for the system, which is given in Equation 
+\ref{eq:hamilthe}
+\begin{equation}\label{eq:hamilthe}
+    \hat{\boldsymbol{H}} = \sum_i^N\hat{h}(i) + \frac{1}{2}\sum_i^N\sum_j^N\frac{1}{|\boldsymbol{r}_i - \boldsymbol{r}_j|}
+\end{equation}
+$\hat{h}(i)$ is a one electron operator containing the potential energy of
+electron $i$ with respect to the nucleus and the kinetic energy of electron $i$.
+The second term accounts for interelectronic interactions. Now we also need a
+wavefunction $\Psi$, which is a Slater determinant. This determinant looks like
+Equation \ref{eq:slater}.
+\begin{equation}\label{eq:slater}
+    \Psi = 1/\sqrt{N!}\begin{vmatrix}
+        \chi_1(\boldsymbol{x}_1) & \cdots & \chi_n(\boldsymbol{x}_1) \\
+        \cdots & \cdots & \cdots \\
+        \chi_1(\boldsymbol{x}_n) & \cdots & \chi_n(\boldsymbol{x}_n)\\
+    \end{vmatrix}
+\end{equation}
+When we look at the individual functions $\chi_i(x_j)$ we can state Equation 
+\ref{eq:spinor}
+
+\begin{equation}\label{eq:spinor}
+    \chi_i(x_j) = \psi_i(\boldsymbol{x}_j)\gamma(\boldsymbol{\omega}_j)
+\end{equation}
+Here $\psi$ is a spatial orbital, $\gamma$ is a spin function, which can be either
+$\alpha$ or $\beta$. Now we have all the ingredients, we can start building the Fock
+matrix. The first step would be to calculate the expectation value of the 
+Hamiltonian over the wave function $\Psi$. First of all we assert that all spin-orbitals
+are normalised, so that $\braket{\chi_i}{\chi_i} = 1$. We will not give the exact 
+derivation here, but jump straight to the end result. This is given in Equation 
+\ref{eq:expect}.
+
+\begin{multline}\label{eq:expect} 
+    \bra{\Psi}\hat{\boldsymbol{H}}\ket{\Psi} = \sum_i^N\bra{\chi_i}\hat{h}(1)\ket{\chi_i} + \frac{1}{2}\sum_i^N\sum_j^N\left(\int\chi_i^*(1)\chi_i(1)\frac{1}{|\boldsymbol{r}_i- \boldsymbol{r}_j|}\chi_j^*(2)\chi_j(2)d1d2 \right.\\

```

Comment:
```suggestion
    \bra{\Psi}\hat{H}\ket{\Psi} = \sum_i^N\bra{\chi_i}\hat{h}(1)\ket{\chi_i} + \frac{1}{2}\sum_i^N\sum_j^N\left(\int\chi_i^*(1)\chi_i(1)\frac{1}{|\boldsymbol{r}_i- \boldsymbol{r}_j|}\chi_j^*(2)\chi_j(2)d1d2 \right.\\

```
Have a look into _split_ and some commands in the physics package, to write this more easily